components/atom/card/src/index.scss

```diff
@@ -7,6 +7,7 @@ $p-atom-card-small: 0 !default;
 $bgc-atom-card-hover: transparent !default;
 $bxsh-atom-card-hover: none !default;
 $c-atom-card-highlight: $c-highlight !default;
+$bgc-atom-card-highlight-hover: $bgc-atom-card-hover !default;
```

Comment:
In order to not break other verticals that could be using Highlight and does not want to have different hover it should be `$c-atom-card-highlight` by default. Then we should override it in our theme, isn't it?

Reply:
The default value should be `transparent !default;` that's the default hover value right now.

Reply:
Currently, `atom/card` is using `$bgc-atom-card-hover` as its `bgc` when on hover on a highlighted card, so using `$c-atom-card-highlight` would actually be a breaking change.

Reply:
What Rafa is pointing is that the value of `$bgc-atom-card-hover` could have changed to other value making that the hover of the highlight could change as well unexpectedly.

So, the default value of the hover should be something that mimics the previous behaviour.

Reply:
I understand it bad.
Now highlight hover removes the highlight, so Jordi does not break it. Just keep the bugfeature 😅 
If we want to persists this bugfeature Jordi is doing it good. If not, Jordi can use `$c-atom-card-highlight` by default in hover.

@midudev what do you think?

Reply:
I would recommend to keep the bugfeature and add a comment about it in the code. :)

Reply:
done